### ISSUE TITLE
gdx-pay-android-googlebilling uses an outdated version of com.android.billingclient

### ISSUE THREAD

**Issue by prineside** [libgdx/gdx-pay#261]
#### Issue details, reproduction steps/code
_Using this library on Android 14 (SDK 34) throws an exception listed below._
_As mentioned [on the Android Developers website](https://developer.android.com/google/play/billing/release-notes#6-0-1), support for SDK 34 is added in version 6.0.1 of the Google Pay Billing library but gdx-pay uses version 6.0.0._
_Quote from the top of that page:_

> 2. If your app is targeting Android 14 or higher, you must update to [PBL 5.2.1](https://developer.android.com/google/play/billing/release-notes#5-2-1) or [PBL 6.0.1](https://developer.android.com/google/play/billing/release-notes#6-0-1) or higher.

#### Version of gdx-pay and/or relevant dependencies
_gdx-pay 1.3.6, which depends on com.android.billingclient:billing:6.0.0 ([here](https://github.com/libgdx/gdx-pay/blob/7bcbcc406301989b3ed0d205647854200e82e55f/gdx-pay-android-googlebilling/build.gradle#L40))_

#### Stacktrace
```java
java.lang.SecurityException: com.prineside.tdi2.debug: One of RECEIVER_EXPORTED or RECEIVER_NOT_EXPORTED should be specified when a receiver isn't being registered exclusively for system broadcasts
    at android.os.Parcel.createExceptionOrNull(Parcel.java:3057)
    at android.os.Parcel.createException(Parcel.java:3041)
    at android.os.Parcel.readException(Parcel.java:3024)
    at android.os.Parcel.readException(Parcel.java:2966)
    at android.app.IActivityManager$Stub$Proxy.registerReceiverWithFeature(IActivityManager.java:5684)
    at android.app.ContextImpl.registerReceiverInternal(ContextImpl.java:1852)
    at android.app.ContextImpl.registerReceiver(ContextImpl.java:1792)
    at android.app.ContextImpl.registerReceiver(ContextImpl.java:1780)
    at android.content.ContextWrapper.registerReceiver(ContextWrapper.java:755)
    at com.android.billingclient.api.zzg.zzc(com.android.billingclient:billing@@6.0.0:1)
    at com.android.billingclient.api.zzh.zze(com.android.billingclient:billing@@6.0.0:3)
    at com.android.billingclient.api.BillingClientImpl.startConnection(com.android.billingclient:billing@@6.0.0:48)
    at com.badlogic.gdx.pay.android.googlebilling.PurchaseManagerGoogleBilling.startServiceConnection(PurchaseManagerGoogleBilling.java:78)
    at com.badlogic.gdx.pay.android.googlebilling.PurchaseManagerGoogleBilling.install(PurchaseManagerGoogleBilling.java:61)
```

#### Please select the affected platforms and payment service implementation
- [ ] Amazon
- [x] googlebilling
- [ ] googlepay
- [ ] apple robovm
- [ ] Huawei

**Comment by keesvandieren** [comment]
Thanks! I will upgrade it to 6.0.1 shortly. I think no code changes are needed?
@prineside  where you able to resolve this by upgrading it to 6.0.1 manually?

**Comment by keesvandieren** [comment]
I'm testing it with the current 6.0.0 on an app with subscriptions on a Pixel 7 Pro simulator. I'm able to start the billing client, able to come into here:
<img width="852" alt="image" src="https://github.com/libgdx/gdx-pay/assets/863966/62d8f013-606c-4529-9636-80f70c3a7007">

Will still do the 6.0.1 upgrade though

**Comment by keesvandieren** [comment]
Resolved in 1.3.7
